tutorials/kafkaexactlyonce/src/main/java/com/example/Application.java

```diff
@@ -16,18 +31,22 @@
 import com.datatorrent.api.annotation.ApplicationAnnotation;
 import com.datatorrent.lib.io.ConsoleOutputOperator;
 
+/**
+ * This application shows the benefits of exactly-once OutputOperators by
+ * comparing the queues of two different Kafka topics using KafkaSinglePortOutputOperator for writing to one topic
+ * and KafkaSinglePortExactlyOnceOutputOperator for writing to an other.
+ *
+ * The Application reads lines from a file on HDFS using LineByLineFileInputOperator
+ * and passes them to the PassthroughFailOperator which emits lines to both of the KafkaOutputOperators. (Kafka 0.9 API)
+ */
+
 @ApplicationAnnotation(name = "KafkaExactlyOnceApp")
 public class Application implements StreamingApplication
 {
 
   @Override
   public void populateDAG(DAG dag, Configuration conf)
   {
-
-    //test
-//    WindowIdInputOperator windowIdInput = dag.addOperator("inputOperator", WindowIdInputOperator.class);
-
-    dag.getAttributes().put(DAG.STREAMING_WINDOW_SIZE_MILLIS, 200);
     //read lines from incoming files in 'inputDirectory' (set in properties.xml)
```

Comment:
Suggest removing comment (all the other operators don't have such a descriptive comment).
In an example, explanatory comments have greater value than in platform or library code, so if you want
to be helpful, suggest a block comment that describes _all_ the operators.